packages/odyssey-storybook/src/components/odyssey-mui/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,112 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Story } from "@storybook/react";
+import { Button, Tooltip } from "@mui/material";
+import { DownloadIcon } from "@okta/odyssey-react-mui";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+
+import TooltipMdx from "./Tooltip.mdx";
+
+export default {
+  title: `MUI Components/Tooltip`,
+  component: Tooltip,
+  parameters: {
+    docs: {
+      page: TooltipMdx,
+    },
+  },
+  argTypes: {
+    content: {
+      control: { type: "object" },
+    },
+    describeChild: {
+      control: { type: "boolean" },
+    },
+    placement: {
+      options: ["top", "right", "bottom", "left"],
+      control: { type: "radio" },
+    },
+    title: {
+      control: { type: "text" },
+    },
+  },
+  decorators: [MuiThemeDecorator],
+};
+
+const Template: Story = (args) => {
+  const {} = args;
+  return (
+    <Tooltip
+      title={args.title}
+      placement={args.placement}
+      describeChild={args.describeChild}
+    >
+      {args.content}
+    </Tooltip>
```

Comment:
We could do `{...args}` if we named the prop `children` rather than `content`. would that be good?

Reply:
Good call on `children`. That's much more appropriate than `content`, and uses the same terminology as the MUI props.

I've been avoiding spreading the args in stories because I like them to be as explicit of examples as possible. Maybe that's not necessary tho.